src/lib/viewers/image/ImageViewer.js

```diff
@@ -17,12 +18,30 @@ class ImageViewer extends ImageBaseViewer {
         this.updatePannability = this.updatePannability.bind(this);
         this.handleImageDownloadError = this.handleImageDownloadError.bind(this);
         this.handleAssetAndRepLoad = this.handleAssetAndRepLoad.bind(this);
+        this.handleZoomEvent = this.handleZoomEvent.bind(this);
+        this.getViewportDimensions = this.getViewportDimensions.bind(this);
+        this.annotationControlsFSM = new AnnotationControlsFSM(
+            this.options.enableAnnotationsImageDiscoverability ? AnnotationState.REGION : AnnotationState.NONE,
```

Comment:
This makes users in explicit region mode by default (region control button is on). I thought we want the image to have the same experience as document (annotations in region mode, but region control button is off). Can you confirm with product that this is the desired behavior?

Reply:
Discussed offline, the desired behavior is that the button is on.